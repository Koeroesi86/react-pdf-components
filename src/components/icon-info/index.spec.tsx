import React from "react";
import { renderToString } from "@react-pdf/renderer";
import { Wrapper } from "@test-helpers";
import IconInfo from "./index";

describe("<IconInfo/>", () => {
  it("should render", async () => {
    const result = await renderToString(
      <Wrapper>
        <IconInfo width={12} height={12} color="#000"/>
      </Wrapper>
    );
    expect(result).toMatchSnapshot();
  });
});
