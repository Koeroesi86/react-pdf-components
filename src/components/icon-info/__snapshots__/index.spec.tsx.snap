// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconInfo/> should render 1`] = `
"%PDF-1.3
%����
8 0 obj
<<
/Type /Page
/Parent 1 0 R
/MediaBox [0 0 595.280029 841.890015]
/Contents 6 0 R
/Resources 7 0 R
/UserUnit 1
>>
endobj
7 0 obj
<<
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
>>
endobj
10 0 obj
(react-pdf)
endobj
11 0 obj
(react-pdf)
endobj
12 0 obj
(D:20210414183756Z)
endobj
9 0 obj
<<
/Producer 10 0 R
/Creator 11 0 R
/CreationDate 12 0 R
>>
endobj
4 0 obj
<<
>>
endobj
3 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Names 2 0 R
/ViewerPreferences 5 0 R
>>
endobj
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [8 0 R]
>>
endobj
2 0 obj
<<
/Dests <<
  /Names [
]
>>
>>
endobj
5 0 obj
<<
/DisplayDocTitle true
>>
endobj
6 0 obj
<<
/Length 352
/Filter /FlateDecode
>>
stream
x�eQ�N�0��+���iS犐8Å���]	!��x&�vW(j<���G��v�]���5ƴ�vv�v��O�gvI���WÃe�����2�9�Fŝ����t��rS>�|Q򒕯w�����|��j���'�~S�O�8Y�W��j�K6��X�mN}�j��d�qa�9F׍�ɞ�怦�dwu�x@)��jH�{R��\\�B��0�H3Y�-Q�k�,=�i9��]h*6+�>�S�O��z�HC���X�U�3FZ�	��$�!o�� >�:fw@���ȿ�e�)�ƕ�h'��Yk��ZRʵ��9�z>�ԻZԄ�p��W�� �z���������f����dc�s��m�ý^�k�^
endstream
endobj
xref
0 13
0000000000 65535 f 
0000000487 00000 n 
0000000544 00000 n 
0000000400 00000 n 
0000000379 00000 n 
0000000591 00000 n 
0000000634 00000 n 
0000000145 00000 n 
0000000015 00000 n 
0000000304 00000 n 
0000000212 00000 n 
0000000240 00000 n 
0000000268 00000 n 
trailer
<<
/Size 13
/Root 3 0 R
/Info 9 0 R
/ID [<89bd153f05caf3c6700333c680a70f52> <89bd153f05caf3c6700333c680a70f52>]
>>
startxref
1058
%%EOF
"
`;
